documentation/EstNumJellies.py

```diff
@@ -63,15 +64,14 @@ def set_frac_ppl_loving_pink(self, frac):
         self.fracPplLovingPink = frac
 
 
-    ## Return the scaling constant so the user can check it if they want.
+    ## A getter for the scaling constant.
```

Comment:
You only need one # after the first line that has ##. I think having ## on subsequent lines might confuse it, but I'm not sure.